### DISCUSSION TITLE
Struggling to get Valheim to show up in the server list / people can't connect via public IP

### DISCUSSION THREAD

**Discussion by ouberlord** [GameServerManagers/LinuxGSM#4077] [Q&A]
I'm just about at wits end with trying to get my local Valheim server to be accessible from the outside world.  I can connect to it locally, so I know the server is running.  I've opened the ports on the router:
![image](https://user-images.githubusercontent.com/18178847/211236156-2f6c4183-35b2-4a4f-a769-1f6ed3ff370c.png)

Here's the output of postdetails: https://termbin.com/lhmm

Of note, it shows "Master server: listed".  However, you can check either in game or via the Steam server browser, and the server does not appear.  That output is clearly a false positive; the server isn't listed, but I don't know how to troubleshoot things further from here.  The OS firewall is not on, and I've used this same server to host other games in the past with no issues.

Does anyone have ideas I can try?

**Comment by makemegit** [comment]
Hi there,

My server is working with these open ports:
2456 tcp and udp
7777 tcp and udp

Please note that people can join my server only if they join it from the game and not from servers (steam menu) with my_address:2456
Hope this will help you.
